quelpa.el

```diff
@@ -274,19 +312,29 @@ local file:
 Installs a single-file package from a local file.  Use the :url
 attribute with an URL like \"file:///path/to/file.el\".
 
+local directory:
+
+Installs a multi-file package from a local directory.  Use
+the :url attribute with an URL like \"file:///path/to/dir\".
+
 remote file:
 
 Installs a single-file package from a remote file.  Use the :url
 attribute with an URL like \"http://domain.tld/path/to/file.el\"."
   (let* ((url (plist-get config :url))
          (version (plist-get config :version))
-         (type (file-name-extension url))
-         (remote-file-name (file-name-nondirectory
-                            (url-filename (url-generic-parse-url url))))
-         (local-path (expand-file-name remote-file-name dir))
+         (urlobj (url-generic-parse-url url))
+         (urltype (url-type urlobj))
+         (urlfile (url-filename urlobj))
+         (remote-file-name (file-name-nondirectory urlfile))
+         (type (if (and (or (null urltype)
```

Comment:
Assuming it's permitted to have no URL protocol in the URL while still having an URL file (which something like `/foo/bar/baz` as URL would lead to), why isn't this mentioned in the docstring?  After all, the README does do that...